packages/renderer/src/lib/deployments/DeploymentColumnConditions.spec.ts

```diff
@@ -37,51 +37,176 @@ function createDeploymentUI(conditions: DeploymentCondition[]) {
 }
 
 test('Expect column styling available', async () => {
-  const deployment = createDeploymentUI([{ type: 'Available', message: 'Running fine' }]);
+  const deployment = createDeploymentUI([
+    { type: 'Available', message: 'Running fine', reason: 'MinimumReplicasAvailable' },
+  ]);
   render(DeploymentColumnConditions, { object: deployment });
 
   const text = screen.getByText('Available');
   expect(text).toBeInTheDocument();
 
-  const dot = text.parentElement?.children[0];
-  expect(dot).toBeInTheDocument();
-  expect(dot).toHaveClass('text-green-600');
+  const svg = text.parentElement?.querySelector('svg');
+  expect(svg).toBeInTheDocument();
+  expect(svg).toHaveClass('text-[var(--pd-status-running)]');
+});
+
+/* Add more tests for
```

Comment:
I don't love this much copypasta b/c someone adding or changing an element in the future might not realize it's here. Not a blocker, but I would remove.

Reply:
woops! those were in-progress comments I was making